services/web/server/src/simcore_service_webserver/director_v2/_controller/_rest_exceptions.py

```diff
@@ -43,10 +44,11 @@ async def _handler_director_service_error_as_503_or_4xx(
     if status_codes_utils.is_5xx_server_error(exception.status):
         # NOTE: All directorv2 5XX are mapped to 503
         status_code = status.HTTP_503_SERVICE_UNAVAILABLE
-        user_msg = (
+        user_msg = user_message(
             # Most likely the director service is down or misconfigured so the user is asked to try again later.
-            "This service is temporarily unavailable. The incident was logged and will be investigated. "
-            + MSG_TRY_AGAIN_OR_SUPPORT
+            "This service is temporarily unavailable. The incident has been logged and will be investigated. "
```

Comment:
Minor: Not sure why it changed from simple past to present perfect? both are correct.

"Was logged" and "has been logged" are both past forms of the verb "to log," but they indicate different timeframes. "Was logged" is the simple past, referring to a completed action in the past. "Has been logged" is the present perfect, indicating an action that started in the past and continues to be relevant to the present, or an action completed at an unspecified time in the past with present relevance.